packages/dapp/package.json

```diff
@@ -20,13 +20,13 @@
     "@transferto/shared": "*",
     "@types/react": "^18.2.15",
     "@widgetbot/react-embed": "^1.7.0",
-    "ethers": "^5.7.2",
+    "ethers": "^6.6.2",
```

Comment:
Should have mentioned this in the task, but we don't need bump ethers. 🙂

Reply:
Alright, left it as 5.7.2